src/core/tree-sitter/loadLanguage.ts

```
@@ -0,0 +1,29 @@
+
+const require = createRequire(import.meta.url);
+
+export async function loadLanguage(langName: string): Promise<Parser.Language> {
+  if (!langName) {
+    throw new Error('Invalid language name');
+  }
+
+  try {
+    const wasmPath = await getWasmPath(langName);
+    return await Parser.Language.load(wasmPath);
+  } catch (error: any) {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)